Libplanet.Net/Swarm.BlockSync.cs

```diff
@@ -476,26 +477,47 @@ in completedBlocks.WithCancellation(cancellationToken))
                         $"is unusable for the the existing chain #0 {g}.");
                 }
 
+                if (!(tipCandidate is { }))
+                {
+                    throw new SwarmException(
+                        $"Tip candidate is null.");
+                }
+
                 // NOTE: All blocks in deltaBlocks will have non-null previousHash.
-                var deltaBlocks = new LinkedList<Block<T>>(new[] { tipCandidate });
+                BlockChainSlice blockChainSlice = new BlockChainSlice(new[] { tipCandidate.Hash });
```

Comment:
For conciseness:

```suggestion
                BlockChainSlice blockChainSlice = new (new[] { tipCandidate.Hash });
```

Or:


```suggestion
                var blockChainSlice = new BlockChainSlice(new[] { tipCandidate.Hash });
```